app/views/spree/shipment_mailer/shipped_email.html.haml

```diff
@@ -15,11 +15,11 @@
     %br
 
 - if @shipment.tracking
-  %p
+  %p{ style: "font-size: 17px;" }
```

Comment:
I think that it would be more consistent to use `p.lead` here so that the formatting is the same as other paragraphs, too.